spec/unit/controllers/runtime/spaces_controller_spec.rb

```diff
@@ -194,6 +194,22 @@ module VCAP::CloudController
       end
     end
 
+    describe 'GET /v2/spaces/:guid/quota_usage' do
+      context 'for an space that does not exist' do
+        it 'returns a 404' do
+          get '/v2/spaces/foobar/quota_usage', {}, admin_headers
+          expect(last_response.status).to eq(404)
+        end
+      end
+      context 'when the user does not have permissions to read' do
+        let(:user) { User.make }
+        it 'returns a 403' do
+          get "/v2/spaces/#{space_one.guid}/quota_usage", {}, headers_for(user)
+          expect(last_response.status).to eq(403)
+        end
+      end
```

Comment:
what about the "posive" context when user has permission and quota exists, we should have a context and test(s) to return it.

Also, add new lines between contexts to keep things easier to read and consistent.